dd-sdk-android-core/build.gradle.kts

```diff
@@ -104,6 +104,8 @@ dependencies {
     implementation(libs.androidXCollection)
     implementation(libs.androidXWorkManager)
 
+    implementation(project(":dd-sdk-android-internal"))
```

Comment:
if we have the core depending on it why do we need it also in the other modules depending on core ? If you need the API in the other modules maybe we could use `api` instead of `implementation` here